ncbi/src/main/java/org/reactome/release/ReactomeConstants.java

```diff
@@ -0,0 +1,6 @@
+package org.reactome.release;
+
+public class ReactomeConstants {
```

Comment:
We already have a `ReactomeJavaConstants` in Guanming's library. Maybe renaming this to `ReactomeURLs` would reduce the potential for confusion?

Reply:
`ReactomeURLConstants`?

Reply:
Changed to `ReactomeURLConstants`!